src/pages/vereniging/activiteiten.jsx

```diff
@@ -1,18 +1,50 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { graphql } from 'gatsby';
 import ActivityCollection from '$/components/activities/ActivityCollection';
 import ContentfulPage from '$/components/layout/ContentfulPage';
 import Markdown from 'markdown-to-jsx';
+import Activity from '$/components/activities/Activity';
+import { Grid } from 'semantic-ui-react';
 
-const Activiteiten = props => {
-  const page = props.data.contentfulPage;
-  return (
-    <ContentfulPage page={page}>
-      <Markdown>{page.content.content}</Markdown>
-      <ActivityCollection count="all" />
-    </ContentfulPage>
-  );
-};
+export default class Activities extends Component {
+  constructor(props) {
+    super(props);
+    this.updateActivities = this.updateActivities.bind(this);
+    this.state = {
+      activities: [],
+    };
+  }
+
+  updateActivities(activities) {
+    this.setState({
+      activities: activities,
+    });
+  }
+
+  render() {
+    const page = this.props.data.contentfulPage;
+    return (
+      <ContentfulPage page={page}>
+        <Markdown>{page.content.content}</Markdown>
+        <ActivityCollection updateActivities={this.updateActivities}>
+          {this.state.activities.length === 0 ? (
+            <p>No activities</p>
+          ) : (
+            <Grid doubling columns={4} centered>
```

Comment:
Because the doubling prop makes it responsive in a nice way. I don't think you get that kind of responsiveness for free with css-grid (you'd have to write your own media queries)

Reply:
We talked about this in a meeting and we decided it's best to use the Semantic UI Grid for grids with homogeneous items and CSS Grid for grids with heterogeneous items like the home page.